static/usr/lib/systemd/system/vdsm-network.service.in

```diff
@@ -9,6 +9,8 @@ Type=oneshot
 EnvironmentFile=-/etc/sysconfig/vdsm
 ExecStartPre=@BINDIR@/vdsm-tool dump-bonding-options
 ExecStart=@BINDIR@/vdsm-tool restore-nets
```

Comment:
Please move it after everything else, this should run as the last thing of this service.

Reply:
Done